servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ProxySelectorTests.java

```diff
@@ -22,7 +22,7 @@
 import java.util.concurrent.TimeUnit;
 
 public class ProxySelectorTests extends TestBase {
-    @Test
+//    @Test
```

Comment:
Use the skip annotation.

Reply:
Yes, I agree - except it is called `@Ignore`. Don't comment out tests, ignore them instead.